roles/sahara/templates/etc/sahara/sahara.conf

```diff
@@ -0,0 +1,23 @@
+[DEFAULT]
+debug = {{ sahara.logging.debug }}
+verbose = true
```

Comment:
There is an attempt at making this a variable, but then this isn't templated here.